CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
-## 23.9.4 2023-07-05
+## X.Y.Z 2023-XX-YY
+### Payments
+* [Fixed] A bug where `mandate_data` was not being properly attached to PayPal SetupIntent's.
 ### PaymentSheet
```

Comment:
@yuki-stripe can you confirm if this change log looks good? It looks like the deferred intent GA was put under the 7/5 release?

Reply:
No it didn't go in the 7/5 release.  Must have been a bad automatic merge with master or my mistake to put it there in the CHANGELOG.  This looks right.